### PR TITLE
Link fix

### PR DESCRIPTION
**Pull request by kolchfa-aws**
Fixes broken links

### Checklist
- [x ] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: _api-reference/snapshots/get-snapshot-status.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ GET _snapshot/my-opensearch-repo/my-first-snapshot/_status
 
 #### Example response
 
-The example that follows corresponds to the request above in the [Example request](#sample-request) section.
+The example that follows corresponds to the request above in the [Example request](#example-request) section.
```
Comment (carolxob) [inline _api-reference/snapshots/get-snapshot-status.md:57]:
```suggestion
The example that follows corresponds to the preceding request in the [Example request](#example-request) section.
```

File: _api-reference/snapshots/restore-snapshot.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ Restores a snapshot of a cluster or specified data streams and indices.
 
 * For information about data streams, see [Data streams]({{site.url}}{{site.baseurl}}/opensearch/data-streams).
 
-If open indices with the same name that you want to restore already exist in the cluster, you must close, delete, or rename the indices. See [Sample Request](#sample-request) for information about renaming an index. See [Close index]({{site.url}}{{site.baseurl}}/api-reference/index-apis/close-index) for information about closing an index.
+If open indices with the same name that you want to restore already exist in the cluster, you must close, delete, or rename the indices. See [Sample Request](#example-request) for information about renaming an index. See [Close index]({{site.url}}{{site.baseurl}}/api-reference/index-apis/close-index) for information about closing an index.
```
Comment (carolxob) [inline _api-reference/snapshots/restore-snapshot.md:17]: Is it safe to change instances of "indices" here to "indexes"?

File: _api-reference/snapshots/verify-snapshot-repository.md [hunk 3]
```diff
@@ -40,7 +40,7 @@ POST /_snapshot/my-opensearch-repo/_verify?timeout=0s&cluster_manager_timeout=50
 
 #### Example response
 
-The example that follows corresponds to the request above in the [Example request](#sample-request) section.
+The example that follows corresponds to the request above in the [Example request](#example-request) section.
```
Comment (carolxob) [inline _api-reference/snapshots/verify-snapshot-repository.md:43]: Same as previous comment above - change "above" to "preceding" (or similar).

**Review by carolxob — APPROVED** [review]
Looks good with a couple of minor suggestions.

**Review by ariamarble — APPROVED** [review]

**Pull request merged**